bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/things/AutomowerHandler.java

```diff
@@ -51,29 +53,39 @@
 import org.openhab.core.thing.binding.ThingHandlerService;
 import org.openhab.core.types.Command;
 import org.openhab.core.types.RefreshType;
+import org.openhab.core.types.Type;
+import org.openhab.core.types.UnDefType;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.gson.Gson;
+
 /**
  * The {@link AutomowerHandler} is responsible for handling commands, which are
  * sent to one of the channels.
  *
  * @author Markus Pfleger - Initial contribution
+ * @author Marcin Czeczko - Added support for planner & calendar data
  */
-@NonNullByDefault
```

Comment:
Did you remove this by intention?

Reply:
I think I did it intentionally that time due to the new class field `mowerState` but apparently I forgot to clean it up. Already fixed it.